workflows/move2kube/README.md

```diff
@@ -5,11 +5,11 @@ This workflow is using https://move2kube.konveyor.io/ to migrate the existing co
 Once the transformation is over, move2kube provides a zip file containing the transformed repo.
 
 ### Design diagram
-![sequence_diagram.svg](https://raw.githubusercontent.com/parodos-dev/serverless-workflows/main/move2kube/sequence_diagram.jpg)
-![design.svg](https://raw.githubusercontent.com/parodos-dev/serverless-workflows/main/move2kube/design.svg)
+![sequence_diagram.svg](https://github.com/parodos-dev/serverless-workflows/blob/main/workflows/move2kube/sequence_diagram.jpg?raw=true)
+![design.svg](https://github.com/parodos-dev/serverless-workflows/blob/main/move2kube/design.svg?raw=true)
```

Comment:
this one also need to be fixed: `https://github.com/parodos-dev/serverless-workflows/blob/main/workflows/move2kube/design.svg?raw=true`

Reply:
Holly molly, I overlooked it... Let me handle it in the next commit